packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -61,7 +61,49 @@ No. You can only set up one custom domain per Okta org.
 
 Yes. When you turn the custom domain on, the Okta domain (for example, `example.okta.com`) still works.
 
-## Validate your TLS certificate
+<ApiLifecycle access="ea" />
+
+## Configure a custom domain via Okta-managed certificates
+
+This method of configuring a custom domain is recommended because Okta will manage your cerificate renewals in imperpetuity, via an integration with Let's Encrypt, a free certificate authority. The certificate procurement process is free, and also faster and easier than configuring a custom domain with your own certificate.
```

Comment:
```suggestion
This method of configuring a custom domain is recommended because Okta manages your certificate renewals in in perpetuity through an integration with Let's Encrypt, which is a free certificate authority. The certificate procurement process is free, and also faster and easier than configuring a custom domain with your own certificate.
```